### PR TITLE
breadcrumbs for command

### PR DESCRIPTION
**Pull request by lusol**
Add breadcrumbs to the commands section 

screen shot 2018-07-02 at 11 59 15 am

issue: I'm not sure if there is one.........

### CODE REVIEW

**Comment by tkyi** [comment]
Here's the issue: https://github.com/screwdriver-cd/screwdriver/issues/1096

File: tests/integration/components/command-section/component-test.js [hunk 1]
```diff
@@ -25,6 +25,6 @@ test('it renders', function (assert) {
 
   this.render(hbs`{{command-section command=mock}}`);
 
-  assert.equal(this.$('header a').text().trim(), 'foo/bar');
+  assert.equal(this.$('header').text().trim(), 'foo / bar');
```
Comment (tkyi) [inline tests/integration/components/command-section/component-test.js]: Why is there added spaces?

File: app/components/command-section/template.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <article class="command">
   <header>
-    {{#link-to "commands.detail" command.namespace command.name}}<h4>{{headerTitle}}</h4>{{/link-to}}
+  <h4> {{#link-to "commands.namespace" command.namespace}}{{command.namespace}}{{/link-to}} / {{#link-to "commands.detail" command.namespace command.name}}{{command.name}}{{/link-to}}</h4>
```
Comment (tkyi) [inline app/components/command-section/template.hbs]: Should also use CSS to space out the text rather than add actual spaces

File: app/router.js [hunk 3]
```diff
@@ -33,7 +33,8 @@ Router.map(function route() {
     this.route('detail', { path: '/namespaces/:namespace/name/:name' });
   });
   this.route('commands', function commandsRoute() {
-    this.route('detail', { path: '/:namespace/:name' });
+    this.route('namespace', { path: '/namespaces/:namespace' });
+    this.route('detail', { path: 'namespaces/:namespace/name/:name' });
```
Comment (tkyi) [inline app/router.js]: missing leading slash (`/`) here

Comment (tkyi) [reply app/router.js]: Wondering if it should be `namespaces/<namespace>/names/<name>` (names vs name)
What do you think @jithin1987 ?

File: app/commands/controller.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+      return params;
+    }
+  }),
+  crumbs: computed('routeParams', {
+    get() {
+      let breadcrumbs = [];
+      let params = this.get('routeParams');
+
+      if (params.namespace || params.detail) {
```
Comment (tkyi) [inline app/commands/controller.js]: I feel like there's a bunch of tests missing for this logic

Comment (lusol) [reply app/commands/controller.js]: hmmmmmm............

File: app/router.js (hunk 3 above)
Comment (lusol) [reply app/router.js]: done

File: app/command/service.js [hunk 5]
```diff
@@ -22,15 +22,21 @@ export default Service.extend({
 
     return this.fetchData(url);
   },
-  getAllCommands() {
-    const url = `${ENV.APP.SDAPI_HOSTNAME}/${ENV.APP.SDAPI_NAMESPACE}/commands`;
+  getAllCommands(namespace) {
+    let url = `${ENV.APP.SDAPI_HOSTNAME}/${ENV.APP.SDAPI_NAMESPACE}/commands`;
```
Comment (tk3fftk) [inline app/command/service.js]: Why is this changed to `let` from `const`?

File: app/router.js (hunk 3 above)
Comment (jithin1987) [reply app/router.js]: it should be consistent with template detail URL

Comment (tkyi) [reply app/router.js]: wondering if we should update the template detail URL to also be `namespaces/<namespace>/names/<name>`

File: app/command/service.js (hunk 5 above)
Comment (lusol) [reply app/command/service.js]: woops, fixed

File: app/commands/controller.js (hunk 4 above)
Comment (lusol) [reply app/commands/controller.js]: nahhhhhh

Comment (jithin1987) [reply app/commands/controller.js]: There should be a corresponding test file ` tests/unit/commands/controller-test.js` to test this

File: tests/unit/commands/controller-test.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  // needs: [],
+});
+
+test('it exists', function (assert) {
+  let controller = this.subject();
+
+  assert.ok(controller);
+});
+
+test('dsdas', function (assert) {
```
Comment (tkyi) [inline tests/unit/commands/controller-test.js]: ?

File: tests/unit/commands/controller-test.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  // needs: [],
+});
+
+test('it exists', function (assert) {
+  let controller = this.subject();
+
+  assert.ok(controller);
+});
+
+test('creates correct breadcrumbs', function (assert) {
```
Comment (tkyi) [inline tests/unit/commands/controller-test.js]: I think test titles usually start with "it"
